### PR TITLE
Chore: Refactore custom-methods

### PR DESCRIPTION
**Pull request by hasanlatch007**
I didn't want to merge the function `sendLove` and `randomReply` with the `@Command` just to keep things clean and neat, if you want me to just move them let me know :).
And I also added `!video` to its own command class but I could also add it to `CustomMessageMethods`

### CODE REVIEW

File: src/common/custom-methods.ts [hunk 1]
```diff
@@ -1,12 +1,213 @@
...
 
 const logger = createYesBotLogger("common", "CustomMethods");
 
-export const sendLove = (message: Message) => {
+@Command({
+  event: DiscordEvent.MESSAGE,
+  trigger: "!video",
+  channelNames: ["welcome-chat"],
+  description:
+    "This handler is to send the Youtube video in the Welcome Chat to new users.",
```
Comment (geisterfurz007) [inline src/common/custom-methods.ts:15]: Commands are not picked up in `src/common`; the distribution only loads from `src/programs`.

File: src/common/custom-methods.ts [hunk 2]
```diff
@@ -1,12 +1,213 @@
...
+})
+class sendVideo implements CommandHandler<DiscordEvent.MESSAGE> {
```
Comment (geisterfurz007) [inline src/common/custom-methods.ts:17]:
```suggestion
class SendVideo implements CommandHandler<DiscordEvent.MESSAGE> {
```

**Review by geisterfurz007 — CHANGES_REQUESTED** [review]

File: src/common/custom-methods.ts (hunk 1 above)
Comment (hasanlatch007) [reply src/common/custom-methods.ts:15]: Testing proved otherwise.

**Comment by hasanlatch007** [comment]
Would rather have this blocked until #453 is merged to avoid merge conflicts after that `custom-methods.ts` can be deleted

**Review by geisterfurz007 — APPROVED** [review]
Nais!

**Pull request merged**